### ISSUE TITLE
Rails 6 compatibility 

### ISSUE THREAD

**Issue by tsrivishnu** [pitr/angular-rails-templates#161]
Is the gem compatible with Rails 6?

**Comment by hiteshranaut** [comment]
not working with rails 6 can you help?
issue I am stuck is where rails 6 need
```
rails (~> 6.0.0) was resolved to 6.0.0, which depends on
      railties (= 6.0.0)
```
but angular-rails-templates required < 6 please help if there is a solution for this?

**Comment by mleone** [comment]
HI @tsrivishnu @hiteshranaut, I was facing the same issue in my app, and this gem might not actually depend on Rails < 6.  I forked this repo and simply changed the rails gem dependency here to allow Rails 6: https://github.com/mleone/angular-rails-templates/commit/c98acac3a74c30ebf3d35521e60cf2966a7d1462

I'm not 100% if this will work for you, but this gem is working properly in my Rails 6 app.

**Comment by mmoll** [comment]
@pitr Could you pull in that commit and release a new gem?

**Comment by pitr** [comment]
released in 1.1.0
